IntelliJ Project/src/main/java/com/globant/musicstore/dao/ClientDAO.java

```diff
@@ -1,9 +1,13 @@
 package com.globant.musicstore.dao;
 
-import com.globant.musicstore.entity.Client;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.globant.musicstore.entity.Client;
 
 public interface ClientDAO extends JpaRepository<Client, Long> {
+	@Query("SELECT client From Client client WHERE client.id IN :id")
+	Client getInvoice(Long id);
```

Comment:
Why is this query here? And what's that naming? If you want to get that query you can just use the included one in the DAO

Reply:
I need that query because of a foreign key in the invoice table. The method name will be changed.